### PR TITLE
Update WebBasic template for beta8. Closes #384.

### LINKED ISSUE #384: ASP.NET Beta8: WebApp template (No Auth)
Samples: http://1drv.ms/1PSX6FB

### PR DESCRIPTION
**Pull request by Spinarooni**

### CODE REVIEW

**Comment by spboyer** [comment]
:+1:

**Comment by peterblazejewicz** [comment]
the `_references.js` is VS artefact: 
http://madskristensen.net/post/the-story-behind-_referencesjs
we haven't included it previously
We should ask VS team in doubt. The VS Code uses inline reference AFAIK.

**Comment by Spinarooni** [comment]
@peterblazejewicz What about the web.config?

**Comment by peterblazejewicz** [comment]
It's also an artefact AFAIK:
https://github.com/OmniSharp/generator-aspnet/pull/394#issuecomment-146301267
But you haven't added that file

**Comment by Spinarooni** [comment]
@peterblazejewicz It was added.  I can remove it though.

File: templates/projects/webbasic/Startup.cs [hunk 1]
```diff
@@ -16,8 +16,9 @@ public class Startup
         public Startup(IHostingEnvironment env, IApplicationEnvironment appEnv)
         {
             // Setup configuration sources.
-            var builder = new ConfigurationBuilder(appEnv.ApplicationBasePath)
-                .AddJsonFile("config.json")
+            var builder = new ConfigurationBuilder()
+                .SetBasePath(appEnv.ApplicationBasePath)
+                .AddJsonFile("appsettings.json")
```
Comment (sayedihashimi) [inline templates/projects/webbasic/Startup.cs]: @rustd I think we are missing a line for `appsettings.{env}.json` here.

**Comment by sayedihashimi** [comment]
OK I think those files are not needed. In VS, `web.config` should be automatically created if missing because of httpplatformhandler work. _references.js.

**Comment by sayedihashimi** [comment]
After thinking about it I think we should add `web.config`. See my comment at https://github.com/OmniSharp/generator-aspnet/pull/394#issuecomment-147052236.

**Pull request merged**